src/js/tabs/pid_tuning.js

```diff
@@ -203,19 +203,15 @@ pid_tuning.initialize = function (callback) {
             }
 
             if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
-                antiGravityGain.attr("min","0.1");
-            }
-
-            if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_45)) {
-                antiGravityGain.attr({ "min": "1", "max": "250", "step": "1" });
+                antiGravityGain.attr({ "min": "0.1", "max": "25.0", "step": "0.1" });
```

Comment:
How this steping value afects to 10.8.0?

Reply:
Good catch (stepping was the same, but not max).